eras/byron/ledger/impl/test/cardano-ledger-byron-test.cabal

```diff
@@ -193,7 +193,7 @@ library
     cardano-crypto-test,
     cardano-crypto-wrapper,
     cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-    cardano-ledger-byron ^>=1.0,
+    cardano-ledger-byron ^>=1.1,
```

Comment:
This change requires a patch version bump, otherwise this package will not get released.
So please bump to `1.5.2.1`

Reply:
I bumped it to `1.6.0.0` because of the transitive `crypton` dependency change.